osclib/accept_command.py

```diff
@@ -137,19 +89,13 @@ def perform(self, project, force=False):
             oldspecs = self.api.get_filelist_for_package(pkgname=req['package'],
                                                          project=self.api.project,
                                                          extension='spec')
-            if 'type' in req and req['type'] == 'delete' and self.api.cdelreq_review:
-                msg += ' and started handling of virtual accept process'
-                print(msg)
-                # Virtually accept the delete request
-                self.virtually_accept_delete(req['id'], req['package'])
-            else:
-                print(msg)
-                change_request_state(self.api.apiurl,
-                                     str(req['id']),
-                                     'accepted',
-                                     message='Accept to %s' % self.api.project,
-                                     force=force)
-                self.create_new_links(self.api.project, req['package'], oldspecs)
+            print(msg)
```

Comment:
if there is only one msg left - just print it

Reply:
PR updated